examples/hello_milvus.py

```diff
@@ -1,90 +1,150 @@
+# We're going to create a collection with 3 fields.
+# +-+------------+------------+------------------+------------------------------+
+# | | field name | field type | other attributes |       field description      |
+# +-+------------+------------+------------------+------------------------------+
+# |1|    "pk"    |    Int64   |  is_primary=True |      "primary field"         |
+# | |            |            |   is_auto=True   |                              |
+# +-+------------+------------+------------------+------------------------------+
+# |2|  "random"  |    Double  |                  |      "a double field"        |
+# +-+------------+------------+------------------+------------------------------+
+# |3|"embeddings"| FloatVector|     dim=8        |  "float vector with dim 8"   |
+# +-+------------+------------+------------------+------------------------------+
+fields = [
+    FieldSchema(name="pk", dtype=DataType.INT64, is_primary=True, auto_id=True),
+    FieldSchema(name="random", dtype=DataType.DOUBLE),
+    FieldSchema(name="embeddings", dtype=DataType.FLOAT_VECTOR, dim=8)
+]
+
+schema = CollectionSchema(fields, "hello_milvus is the simplest demo to introduce the APIs")
+
+print(fmt.format("Create collection"))
+hello_milvus = Collection("hello_milvus", schema)
+
+################################################################################
+# 3. insert entites
+# We are going to insert 3000 number of rows entities into hello_milvus
+# The entites are orgnized in fields.
+#
+# The returned ids if insert() method are
+# - automatically generated by Milvus if auto_id=True in schema
+# - primary key field from the entities
+print(fmt.format("Start inserting entities"))
+entities = [
+    # ommit the pk field because `is_auto` is set to True
+    [float(random.randrange(-20, -10)) for _ in range(3000)],  # field random
+    [[random.random() for _ in range(8)] for _ in range(3000)],  # field embeddings
+]
+
+ids = hello_milvus.insert(entities)
+
+print(f"Number of entities in Milvus: {hello_milvus.num_entities}")  # check the num_entites
+
+################################################################################
+# 4. create index
+# We are going to create an IVF_FLAT index for hello_milvus collection.
+# create_index() can only be applied to `FloatVector` and `BinaryVector` fields.
+print(fmt.format("Start Creating index"))
+index = {
+    "index_type": "IVF_FLAT",
+    "metric_type": "L2",
+    "params": {"nlist": 128},
+}
+
+hello_milvus.create_index("embeddings", index)
+
+################################################################################
+# 5. search, query, and hybrid search
+# After the inserting and indexing, you are able to do some querys.
+# You can do
+# - search based on vector similarity
+# - query based on scalar filtering(boolean, int, etc.)
+# - hybrid search based on vector similarity and scalar filtering.
+#
+# Before you do these searchs, the data of `hello_milvus` need to be loaded to memory
```

Comment:
Before conducting a search or a query, you need to load the data in `hello_milvus` into memory.